modules/nf-core/chopper/meta.yml

```diff
@@ -0,0 +1,45 @@
+name: "chopper"
+description: Filter and trim long read data.
+keywords:
+  - filter
+  - trimming
+  - fastq
+  - nanopore
+  - qc
+tools:
+  - "chopper":
```

Comment:
OK next suggestion is going to be using you as a guinea pig ;)

Reply:
should be fixed now, squashed the commit into the previous one and force pushed onto the chopper branch to have a nicer history